dfget/core/downloader/p2p_downloader/power_client.go

```diff
@@ -38,6 +38,12 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	// downloadPieceTimeoutSeconds specifies the timeout for piece downloading.
+	// If the actual execution time exceeds this threshold, a warning will be thrown.
+	downloadPieceTimeoutSeconds = 2.0
```

Comment:
I would like to see it as `downloadPieceTimeout = 2.0 * time.Second`
Actually we usually directly use duration rather than digital number. Since it would be more readable.

Reply:
Done